onboarding/students-interns.md

```diff
@@ -15,11 +15,11 @@ get familiar with (ask your supervisor if you are unsure).
 
 - [Spinal Cord Toolbox course](https://spinalcordtoolbox.com/user_section/courses.html) (pick the latest course)
 - [](#neuroimaging-analysis)
-- [mriquestions.com](https://mriquestions.com/index.html)
 
 ## AI and deep learning
 
 
 
 ## MRI physics
+- [Bibliography/MRI](https://intranet.neuro.polymtl.ca/bibliography/mri.html#general)
```

Comment:
It is better practice to use target instead of relative links to the built HTML page, in case the page moves to another directory in the future (which will yield a broken link).

See example of code that defines the target:
https://github.com/neuropoly/intranet.neuro.polymtl.ca/blob/d2d2daeae6b3e8e1cfe44c4f66d32f89afd41259/geek-tips/git.md?plain=1#L2

And how to call the target:
https://github.com/neuropoly/intranet.neuro.polymtl.ca/blob/d2d2daeae6b3e8e1cfe44c4f66d32f89afd41259/onboarding/students-interns.md?plain=1#L12

Reply:
I was not familiar with this target link functionality. Did I implement it correctly in my latest commit?

Reply:
Yes! I verified it by checking your branch locally and building the website: 

![image](https://github.com/neuropoly/intranet.neuro.polymtl.ca/assets/2482071/3a14a395-82ab-474d-b0a9-b7fba77588dc)

The link works well. Nice job!

However, we should probably redirect to a more 'curated' headings of the mri bibliography (see [conversation from slack](https://neuropoly.slack.com/archives/C034UD4QW/p1713800053491479))